Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -28,6 +28,5 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     double current_;
     double ampHoursUsed_;
     QTime time_;
```

Comment:
Try and be a bit more specific with these naming, ie is this the time of the "Present time" data point, or the time of the "Previous" data point? Same goes for the current.